ush/python/pygfs/task/land_analysis.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python3
+
+import os
+from logging import getLogger
+from typing import Dict, List
+from pprint import pformat
+
+from pygw.attrdict import AttrDict
+from pygw.file_utils import FileHandler
+from pygw.timetools import to_fv3time, to_YMD, to_YMDH
+from pygw.fsutils import rm_p
+from pygw.yaml_file import parse_j2yaml
+from pygw.jinja import Jinja
+from pygw.logger import logit
+from pygw.executable import Executable
+from pygw.exceptions import WorkflowException
+from pygfs.task.analysis import Analysis
+
+logger = getLogger(__name__.split('.')[-1])
+
+
+class LandAnalysis(Analysis):
+    """
+    Class for global land analysis tasks
+    """
+    @logit(logger, name="LandAnalysis")
+    def __init__(self, config):
+        super().__init__(config)
+
+        # Create a local dictionary that is repeatedly used across this class
+        local_dict = AttrDict(
+            {
+                'OPREFIX': f"{self.runtime_config.RUN}.t{self.runtime_config.cyc:02d}z.",
+            }
+        )
+
+        # task_config is everything that this task should need
+        self.task_config = AttrDict(**self.config, **self.runtime_config, **local_dict)
+
+    @logit(logger)
+    def prepare_IMS(self: Analysis) -> None:
+        """Prepare the IMS data for a global land analysis
+
+        This method will prepare IMS data for a global land analysis using JEDI.
+        This includes:
+        - staging model backgrounds
+        - generating a card file for IMS processing
```

Comment:
what is a card file?

Reply:
No idea. I assume it is the output from `calcfIMS.exe`.  @jiaruidong2017 would know for sure.

Reply:
I think this probably means `generating a namelist for IMS processing` ?

Reply:
possible.  I'll update the comment on Monday with the hash when more reviews come in.

Reply:
Yes, it is namelist for IMS processing